question_2.rb

```diff
@@ -0,0 +1,15 @@
+# Taking user input
+puts 'Enter Grade : '
+grade = gets.chomp.to_i
+
+
+# Conditionals statements to determine school level
+if grade >= 1 and grade <= 5
```

Comment:
Also if your grade is less than 0 then it should return College, please handle validation.